app/gui/src/dashboard/components/dashboard/AssetRow.tsx

```diff
@@ -353,6 +354,18 @@ export function RealAssetInternalRow(props: RealAssetRowInternalProps) {
     }
   }, [grabKeyboardFocusRef, isKeyboardSelected, asset])
 
+  const setDirectoryId = useSetCurrentDirectoryId()
+
+  const dragDelayProps = useDragDelayAction(
+    asset.type === backendModule.AssetType.directory ?
+      () => {
+        React.startTransition(() => {
```

Comment:
it should be `startNavigation` instead: https://github.com/enso-org/enso/blob/develop/app/gui/src/dashboard/components/dashboard/AssetRow.tsx#L232